src/algebraic_topology/simplicial_type.lean

```diff
@@ -0,0 +1,74 @@
+/-
+Copyright (c) 2021 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Scott Morrison
+-/
+import algebraic_topology.simplicial_object
+import category_theory.yoneda
+
+/-!
+A simplicial type (usually called a simplicial set, when using set theoretic foundations)
+is just a simplicial object in `Type`, i.e. a `Type`-valued presheaf on the simplex category.
+
+We define the standard simplices as simplicial types, and their boundaries and horns.
+-/
+
+universes v u
+
+open category_theory
+
+/-- The category of simplicial types.
+This is the category of contravariant functors from
+`simplex_category` to `Type u`. -/
+@[derive large_category]
+def sType : Type (u+1) := simplicial_object (Type u)
+
+namespace sType
+
+/-- The `n`-th standard simplex associated with a nonempty finite linear order `n`
+is the Yoneda embedding of `n`. -/
+def standard_simplex : simplex_category ⥤ sType := yoneda
+
+instance : inhabited sType := ⟨standard_simplex.obj (0 : ℕ)⟩
+
+/-- The `m`-simplices of the `n`-th standard simplex are
+the monotone maps from `fin (m+1)` to `fin (n+1)`. -/
+def as_preorder_hom {n} {m} (α : (standard_simplex.obj n).obj m) :
+  preorder_hom (fin (m.unop+1)) (fin (n+1)) := α
+
+/-- The boundary of the `n`-th standard simplex consists of
+all `m`-simplices of `standard_simplex n` that are not surjective
+(when viewed as monotone function `m → n`). -/
```

Comment:
Probably worth remarking somewhere that this is the simplicial set usually denoted by something like `∂Δ[n]`, even if we don't necessarily want to fix a Lean notation for it yet.

Reply:
Done, and added the (localized) notation.